create_tag.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python3
+
+"""Release bot"""
+
+import argparse
+import subprocess
+import sys
+import os
+import time
+import logging
+from datetime import date
+from ghapi.all import GhApi
+
+
+class fg:  # pylint: disable=too-few-public-methods
+    """Set of constants to print colored output in the terminal"""
+    BOLD = '\033[1m'  # bold
+    OK = '\033[32m'  # green
+    INFO = '\033[33m'  # yellow
+    ERROR = '\033[31m'  # red
+    RESET = '\033[0m'  # reset
+
+
+def msg_error(body):
+    """Print error and exit"""
+    print(f"{fg.ERROR}{fg.BOLD}Error:{fg.RESET} {body}")
+    sys.exit(1)
+
+
+def msg_info(body):
+    """Print info message"""
+    print(f"{fg.INFO}{fg.BOLD}Info:{fg.RESET} {body}")
+
+
+def msg_ok(body):
+    """Print ok status message"""
+    print(f"{fg.OK}{fg.BOLD}OK:{fg.RESET} {body}")
+
+
+def run_command(argv):
+    """Run a shellcommand and return stdout"""
+    result = subprocess.run(  # pylint: disable=subprocess-run-check
+        argv,
+        capture_output=True,
+        text=True,
+        encoding='utf-8')
+
+    if result.returncode == 0:
+        ret = result.stdout.strip()
+    else:
+        ret = result.stderr.strip()
+
+    return ret
+
+
+def autoincrement_version(latest_tag):
+    """Bump the version of the latest git tag by 1"""
+    if latest_tag == "":
+        msg_info("There are no tags yet in this repository.")
+        version = "1"
+    elif "." in latest_tag:
+        version = latest_tag.replace("v", "").split(".")[0] + "." + str(int(latest_tag[-1]) + 1)
+    else:
+        version = int(latest_tag.replace("v", "")) + 1
+    return version
+
+
+def list_prs_for_hash(args, api, repo, commit_hash):
+    """Get pull request for a given commit hash"""
+    query = f'{commit_hash} type:pr is:merged base:{args.base} repo:osbuild/{repo}'
+    try:
+        res = api.search.issues_and_pull_requests(q=query, per_page=20)
+    except:
+        msg_info(f"Couldn't get PR infos for {commit_hash}.")
+        res = None
+
+    if res is not None:
+        items = res["items"]
+
+        if len(items) == 1:
+            ret = items[0]
+        else:
+            msg_info(f"There are {len(items)} pull requests associated with {commit_hash} - skipping...")
+            for item in items:
+                msg_info(f"{item.html_url}")
+            ret = None
+    else:
+        ret = None
+
+    return ret
+
+
+def get_pullrequest_infos(args, repo, hashes):
+    """Fetch the titles of all related pull requests"""
+    logging.debug("Collect pull request titles...")
+    api = GhApi(repo=repo, owner='ochosi', token=args.token)
+    summaries = []
+    i = 0
+
+    for commit_hash in hashes:
```

Comment:
```suggestion
    for i, commit_hash in enumerate(hashes):
```

is slightly more pythonic.